### DISCUSSION TITLE
Fuel is missing

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4306] [General]
Hi developers and simmers,

flying from PAFA to UHPP with full tanks of fuel. After 1 hour flight, fuel is missing ... Round about 19 tons during takeoff, used 3,3 tons after 60 minutes... where is the delta ? :-p ...

![grafik](https://user-images.githubusercontent.com/76531893/113679320-ee8bee80-96bf-11eb-8f4c-a9f4d0fc87b0.png)

Greets Klaus

**Comment by CptBayer** [comment]
Hi again,

the problem seems to be the center tank. I had an refuel in the air:-P after the center tank was empty again. Set the pumps from the center tank to off and now I will check what happend ..

![grafik](https://user-images.githubusercontent.com/76531893/113683296-55130b80-96c4-11eb-9fec-ee3a72946514.png)

... no loss of fuel with pumps off .....

![grafik](https://user-images.githubusercontent.com/76531893/113702404-bf827680-96d9-11eb-9334-9347d7b9e4ea.png)


Greets Klaus

**Comment by ExampleWasTaken** [comment]
Are you on experimental?

**Reply by CptBayer** [reply]
oh sorry.... Yes !!!

**Reply by ExampleWasTaken** [reply]
It's a known bug on experimental.

**Reply by CptBayer** [reply]
thanx ..... ! Is it also a known issue, that loaded flightplans are genrating massiv stutter? If not, i will generate an issue ..

Greets Klaus

**Reply by ExampleWasTaken** [reply]
Yes it is known and expected to improve with the new flight plan manager.

**Reply by ExampleWasTaken** [reply]
Should your question be answered, I would like to ask you to mark it as such so it gets removed from the discussions front page and other issues can rise higher. 
Thanks

**Reply by CptBayer** [reply]
thanx, it is answered..... where can I mark the thread answered?

**Reply by ExampleWasTaken** [reply]
Never mind I just realized that the category is General. Doesn't work here so it's all good :)
